src/core/include/iDynTree/Core/SpatialInertia.h

```diff
@@ -84,14 +99,15 @@ namespace iDynTree
         Wrench biasWrench(const Twist & V) const;
 
         /**
-         * Return the derivative of the bias wrench with respect to the link twist.
+         * @brief Return the derivative of the bias wrench with respect to the link 6D velocity.
          *
-         * Defining \f$ M \f$ as this inertia, return the derivative
-         * with respect to V  of the bias wrench V.cross(M*V).
+         * Defining \f$ M \in \mathbb{R}^{6 \times 6} \f$ as this inertia, return the derivative
+         * with respect to \f$ \mathrm{v} = \begin{bmatrix} v \\ \omega \end{bmatrix} \in \mathbb{R}^6 \f$
```

Comment:
In the documentation of method `biasWrench(const Twist& v)`, there is a missing identity matrix 1_3 in the 6d inertia matrix defintion.

Reply:
The 6D link velocity is denoted with capital V in that method, which is different from the small v notation used in this method.

Reply:
I am  not getting this comment.

Reply:
Throughout the documentation $m$ is denoted as the mass of the rigid body. So when denoted in the 6D inertia matrix, shouldnt it wrtten as $m1_{3}$, where $1_{3}$ is a $3 \times 3$ identity matrix.

Correct me if I'm wrong.